components/at_sum_members_mention/index.tsx

```diff
@@ -0,0 +1,42 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React, {useState} from 'react';
+
+import NotificationFromMembersModal from './notifcation_from_members_modal';
+
+type Props = {
+    postId: string;
+    text: string;
+    userIds: string[];
+    messageMetadata: Record<string, string>;
+}
+
+function AtSumOfMembersMention(props: Props) {
+    const [show, setShow] = useState(false);
+    const closeModal = () => {
+        setShow(false);
+    };
+
+    return (
+        <>
+            <NotificationFromMembersModal
```

Comment:
@neallred it might be better to leave it like so. Several messages could have a sum say one message reads `5 members...` and another `8 members...`, when they both read from redux state, they will both render the modal.

Reply:
nvm... I wasn't closing the right modal